lib/config-all.json

```diff
@@ -12,12 +12,10 @@
     "hasEvent",
     "html5shiv",
     "html5printshiv",
-    "load",
     "mq",
     "prefixed",
     "prefixes",
     "prefixedCSS",
-    "setClasses",
```

Comment:
why is that gone?

and could you document those changes (in the options) in the README too?